website/layouts/_default/sidemenu.html

```diff
@@ -8,25 +8,25 @@
     <h5>Thanos</h5>
     <div class="list-group list-group-flush">
         {{ range .Site.Menus.thanos }}
-            <a class="list-group-item list-group-item-action list-group-item-thanos py-1" href="{{ .URL }}">{{ .Title }}</a>
+            <a class="list-group-item list-group-item-action list-group-item-thanos py-1" href="{{ .URL }}">&nbsp;&nbsp;&nbsp;&nbsp;{{ .Title }}</a>
```

Comment:
I'm not expert on CSS, but would it be possible to add spaces via `class` or `style` instead of `&nbsp;` ?

Reply:
hm... I think we lost this change somehow: https://github.com/improbable-eng/thanos/pull/807/commits/ff55d5ac489b5626a69f28f05f83821fcbfdfd40

Reply:
Wait, are you reviewing correct version? Refresh @povilasv that was fixed (:

Reply:
ahh :dancing_men: